main.tf

```diff
@@ -50,14 +44,12 @@ resource "aws_vpn_connection" "this" {
 }
 
 resource "aws_vpn_connection_route" "this" {
-  count = var.create_vpn_connection ? length(var.destination_cidr_blocks) : 0
```

Comment:
Still need to count over destination cidr blocks

Reply:
`count = length(var.destination_cidr_blocks)` ?